framework/generate_doc.mjs

```diff
@@ -0,0 +1,116 @@
+import * as fs from 'fs';
+import data from './.jsii.tabl.json' assert { type: 'json' };
+
+// select only snippets for readme files
+const snippets = Object.fromEntries(
+    Object.entries(data.snippets)
+        .filter(([_,v]) => v.location.api.api === 'moduleReadme')
+);
+
+const linksMapping = {
+    '../storage/README.md#datalakestorage' : '../02-Storage/03-data-lake-storage.mdx',
```

Comment:
Any way to auto-generate this mapping? Or, how to ensure we can properly maintain it manually?

Reply:
no way, that's the only thing I could not automate... but if links are not good, docusaurus fails so there is at least a guardrail